addons/l10n_fr_pos_cert/static/src/js/Popups/ClosePosPopup.js

```diff
@@ -0,0 +1,39 @@
+odoo.define('l10n_fr_pos_cert.ClosePosPopup', function (require) {
+    'use strict';
+
+    const ClosePosPopup = require('point_of_sale.ClosePosPopup');
+    const Registries = require('point_of_sale.Registries');
+
+    const PosFrCertClosePopup = (ClosePosPopup) =>
+        class extends ClosePosPopup {
+            getLimitDate() {
+                let limitDate = new Date(this.env.pos.pos_session.start_at);
+                limitDate.setDate(limitDate.getDate() + 1);
+                return limitDate;
+            }
+            canCloseSession() {
+                let canClose = true;
+                if (this.env.pos.is_french_country() && this.env.pos.pos_session.start_at) {
+                    const now = Date.now();
+                    const limitDate = this.getLimitDate();
+                    if (limitDate < now) {
+                        canClose = !this.env.pos.config.module_pos_hr || this.isManager;
```

Comment:
It is to prevent a non-manager to be able to close the session in this particular following use case:
- "Authorized Employees" is activated and a non-manager is connected to the session
- Let the session open for 24 hours 
- After 24 hours, refresh the session page
The closing control popup will be displayed and the non-manager user will be able to close the session.

It's the only reason why I've add this check.

Reply:
Normally, it is not possible for an user without the right to close to display the closing control popup.
But here, it will be showed for every users when the session is outdated.
Also, the closing control popup is displayed before the cashier selection, so no cashier is selected during the check.

Reply:
@kitan191 maybe I am wrong, but I think normal user can close POS, they just cannot have a difference between what is counted and not counted.

Reply:
Ah ok.
I thought it was not allowed for a normal user because the close button is not displayed for him when "Authorized Employees" is activated.

I've removed the condition.